### PR TITLE
Fix docstring for generate_speech()

### LINKED ISSUE #888: Incorrect Docstring
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to try to find a similar issue and didn't find one.
- [X] I searched the Marvin documentation for this issue.

### Bug summary

Incorrect docstring written for `generate_speech` function. Can be checked [here](https://github.com/PrefectHQ/marvin/blob/d3e8e166d99b9d1a30a75748135b33dee8251250/src/marvin/ai/audio.py#L25).

### Reproduction

```python3
Can be see in code for `generate_speech()`.
```

### Error

```
    """
    Generates an image based on a provided prompt template.

    This function uses the DALL-E API to generate an image based on a provided
    prompt template. The function supports additional arguments for the prompt
    and the model.

    Args:
        prompt_template (str): The template for the prompt.
        prompt_kwargs (dict, optional): Additional keyword arguments for the
            prompt. Defaults to None.
        model_kwargs (dict, optional): Additional keyword arguments for the
            language model. Defaults to None.

    Returns:
        ImagesResponse: The response from the DALL-E API, which includes the
            generated image.
    """
```

### Versions

```Text
2.3.1
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Dev-Khant**
Fixes #888

### CODE REVIEW

**Comment by Dev-Khant** [comment]
@zzstoatzz Please review it. Thanks.

File: src/marvin/ai/audio.py [hunk 1]
```diff
@@ -22,9 +22,9 @@ async def generate_speech(
     model_kwargs: Optional[dict[str, Any]] = None,
 ) -> Audio:
     """
-    Generates an image based on a provided prompt template.
+    Generates a speech based on a provided prompt template.
```
Comment (zzstoatzz) [inline src/marvin/ai/audio.py:25]:
```suggestion
    Generates speech based on a provided prompt template.
```

File: src/marvin/ai/audio.py [hunk 2]
```diff
@@ -22,9 +22,9 @@ async def generate_speech(
...
 
-    This function uses the DALL-E API to generate an image based on a provided
+    This function uses the OpenAI Audio API to generate an speech based on a provided
```
Comment (zzstoatzz) [inline src/marvin/ai/audio.py:27]:
```suggestion
    This function uses the OpenAI Audio API to generate speech based on a provided
```

**Review by zzstoatzz — COMMENTED** [review]
thanks @Dev-Khant - just a couple nits

**Review by zzstoatzz — APPROVED** [review]

**Pull request merged**